docs/user-guide/data.rst

```diff
@@ -10,13 +10,44 @@ Overview
 Data Description
 ----------------
 
-+----------+---------------------------------------+---------------------------------------+
-| Level    | Product                               | Description                           |      
-+==========+=======================================+=======================================+
-| 1        | Vector magnetic field from 3 sensors. | CCSDS, each packet is 4-sec long      |
-|          |                                       | at 10 Hz rate, 3-axis magnetic        |
-|          |                                       | field components for all three sensors|
-+----------+---------------------------------------+---------------------------------------+
++----------+---------------------------------------+---------------------------------------------------------------------------------------+
+| Level    | Product                               | Description                                                                           |      
++==========+=======================================+=======================================================================================+
+| 1        | H+ Counts                             | 32 Energies x 8 Deflections (256 HV step) x 16 Anodes every 0.87s (may be compressed) |
```

Comment:
Technically, there are 1024 steps in the sweep. We use microstepping, where every 4 steps are summed together, resulting in 256 steps for each anode. The data product is an array of size 4096 bytes.